### PR TITLE
Update `dask_cuda` usage in `SSHCluster` docstring

### PR DESCRIPTION
**Pull request by jrbourbeau**
cc @quasiben @pentschev

Closes https://github.com/dask/distributed/issues/4814

### CODE REVIEW

File: distributed/deploy/ssh.py [hunk 1]
```diff
@@ -295,14 +295,14 @@ def SSHCluster(
...
-    ``dask-cuda-worker`` command from the ``dask-cuda`` project.
+    ``CUDAWorker`` worker from the ``dask-cuda`` project.
 
     >>> from dask.distributed import Client, SSHCluster
     >>> cluster = SSHCluster(
     ...     ["localhost", "hostwithgpus", "anothergpuhost"],
     ...     connect_options={"known_hosts": None},
     ...     scheduler_options={"port": 0, "dashboard_address": ":8797"},
-    ...     worker_module='dask_cuda.dask_cuda_worker')
+    ...     worker_module="dask_cuda.CUDAWorker")
```
Comment (pentschev) [inline distributed/deploy/ssh.py:305]:
```suggestion
    ...     worker_module="dask_cuda.cli.dask_cuda_worker")
```

Comment (pentschev) [reply distributed/deploy/ssh.py:305]: `dask_cuda.CUDAWorker` is the class that creates the `Nanny` object. I think what's expected here is the CLI interface, which is what I suggested above.

File: distributed/deploy/ssh.py [hunk 2]
```diff
@@ -295,14 +295,14 @@ def SSHCluster(
     >>> client = Client(cluster)
 
     An example using a different worker module, in particular the
-    ``dask-cuda-worker`` command from the ``dask-cuda`` project.
+    ``CUDAWorker`` worker from the ``dask-cuda`` project.
```
Comment (pentschev) [inline distributed/deploy/ssh.py:298]: We could revert this change as per my suggestion below.

**Review by pentschev — CHANGES_REQUESTED** [review]
Thanks @jrbourbeau for doing this, I added a few comments.

File: distributed/deploy/ssh.py (hunk 1 above)
Comment (jrbourbeau) [reply distributed/deploy/ssh.py:305]: Thanks @pentschev -- I thought this was a `worker_class=` keyword which is more commonly used than `worker_module=`. Updating now

Comment (pentschev) [reply distributed/deploy/ssh.py]: Yeah, it can feel a bit counterintuitive as in most cases we're working with Python without a CLI, but to launch over SSH this takes a pretty straightforward path of just launching the command remotely.

**Review by pentschev — APPROVED** [review]
LGTM, thanks @jrbourbeau !

**Comment by jrbourbeau** [comment]
Thanks for reviewing @pentschev!
